samples/host_verify/README.md

```diff
@@ -66,4 +66,6 @@ When you see the following message displayed on the screen, it means you have su
 Usage:
   ./host_verify -r <report_file>
   ./host_verify -c <certificate_file>
```

Comment:
I still don't understand this sample.  Line 11 says it can be used to verify a remote report.  But line 18 looks like it will only work for a local enclave not a remote one.  How does one get a <report_file> or a <certificate_file> for a remote enclave?  maybe line 11 is wrong and this is just for local testing?

Reply:
> maybe line 11 is wrong and this is just for local testing?

My understanding is that it is just for local testing. The utility can be used to verify any remote reports.